mmv1/third_party/terraform/website/docs/r/compute_instance_group_manager.html.markdown

```diff
@@ -149,16 +149,19 @@ group. You can specify only one value. Structure is [documented below](#nested_a
 
 ```hcl
 update_policy {
-  type                  = "PROACTIVE"
-  minimal_action        = "REPLACE"
-  max_surge_percent     = 20
-  max_unavailable_fixed = 2
-  min_ready_sec         = 50
-  replacement_method    = "RECREATE"
+  type                           = "PROACTIVE"
+  minimal_action                 = "REPLACE"
+  most_disruptive_allowed_action = "REPLACE"
+  max_surge_percent              = 20
+  max_unavailable_fixed          = 2
+  min_ready_sec                  = 50
+  replacement_method             = "RECREATE"
 }
 ```
 
-* `minimal_action` - (Required) - Minimal action to be taken on an instance. You can specify either `RESTART` to restart existing instances or `REPLACE` to delete and create new instances from the target template. If you specify a `RESTART`, the Updater will attempt to perform that action only. However, if the Updater determines that the minimal action you specify is not enough to perform the update, it might perform a more disruptive action.
+* `minimal_action` - (Required) - Minimal action to be taken on an instance. You can specify either `REFRESH` to update without stopping instances, `RESTART` to restart existing instances or `REPLACE` to delete and create new instances from the target template. If you specify a `REFRESH`, the Updater will attempt to perform that action only. However, if the Updater determines that the minimal action you specify is not enough to perform the update, it might perform a more disruptive action.
```

Comment:
The docs still state this is required

Reply:
Fixed, Thank you for catching.

Reply:
I reverted this change. Lets decouple this discussion from this PR.